lib/generators/rodauth/templates/app/views/rodauth/tailwind/logout.html.erb

```diff
@@ -0,0 +1,14 @@
+<% content_for :title, rodauth.logout_page_title %>
+<div class="flex justify-center">
+  <%= form_with url: rodauth.logout_path, method: :post, data: { turbo: false } do |form| %>
+    <% if rodauth.features.include?(:active_sessions) %>
+      <div class="py-3">
+        <%= form.check_box rodauth.global_logout_param, id: "global-logout", class: "appearance-none checked:bg-blue-500" %>
+        <%= form.label "global-logout", rodauth.global_logout_label, class: "inline-block text-gray-800" %>
+      </div>
+    <% end %>
+    <div class="py-3">
+      <%= form.submit rodauth.logout_button, class: "px-8 py-2 font-semibold rounded-md flex justify-center py-2 px-4 border border-transparent text-sm font-medium rounded-md text-white bg-blue-600 hover:bg-blue-700 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-blue-500 dark:bg-emerald-400 dark:text-gray-900" %>
+    </div>
+  </div>
+<% end %>
```

Comment:
All files should have a trailing newline, you'll see GitHub warning you in the diff when it's missing 😉

Reply:
Fixed with [1a8c11f](https://github.com/janko/rodauth-rails/pull/114/commits/1a8c11f5fb367862b7b2621f0f446fc0ec614eb5)